### DISCUSSION TITLE
Capstone DA (Flask Dashbord UI)

### DISCUSSION THREAD

**Discussion by MiftahulLabiib** [teamalgoritma/community#617] [Q&A] [answered]
Saya ingin bertanya, jadi saya sudah melengkapi semua code dan sudah saya pindahkan di vscode detelah di run berhasil, tapi saat buka dashboard nya gambar graph nya ngga ke load/ngga muncul padahal sudah dilakukan sesuai instruksi video.

**Comment by Yusraf002** [comment]
Dear @MiftahulLabiib 
Terima kasih atas pertanyaannya pak, tim kami akan segera coba bantu atau carikan referensi ya pak.

**Comment by divakartika** [comment, marked as answer]
Dear @MiftahulLabiib ,

Dapat dipastikan apakah seluruh bagian-bagian kosong dalam file app.py sudah terisi?
